theme/gen.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	bundleFont(fontFace, "Bold", f)
 	bundleFont(fontFace, "Italic", f)
 	bundleFont(fontFace, "BoldItalic", f)
-	bundleFont("NotoMono", "Regular", f)
+	bundleFont("DejaVuSansMono", "Powerline", f)
```

Comment:
Does this TextGrid PR require the new font or can it be added separately? Is there an issue with NotoMono?

Reply:
Good point. As my test app was a terminal the NotoMono didn't really cut the mustard.
Powerline was needed to make this good enough IMHO.
The move from noto mono to dejavu sans was just a personal choice that I forgot about.
There is a noto powerline if you'd prefer.

Reply:
If DejaVu is a better font we should use it, but maybe it should be added in its own PR. If further down the road we decide we don't like it we can revert it without reverting TextGrid.

Reply:
True, but reverts are done at a commit level not a PR level, so creating a new PR makes no difference in that regard :)